packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -498,6 +498,7 @@ export default decorate([
               autoComplete='new-password'
               className='form-control'
               name='encryptionKey'
+              placeholder='Enter your encryption key here (32 characters)'
```

Comment:
All text content should be internationalizable. For this, you need to use 
```suggestion
              placeholder='{_('remoteEncryptionKeyPlaceholder')}'
```
and then define the key `remoteEncryptionKeyPlaceholder` (or any better name )  in `packages/xo-web/src/common/intl/messages.js` somewhere around the other keys used in this page ( line 548 - 565)